### ISSUE TITLE
otp invalid

### ISSUE THREAD

**Issue by aisamuel** [harryanand060/Django_DRF#1]
I kept getting "invalid otp" despite entering the correct otp sent to my mail
[
    {
        "status": false,
        "data": null,
        "message": **"invalid otp**",
        "status_code": 401
    },
    401
]

**Comment by harryanand060** [comment]
Thanks aisamuel for this issue. i will check and come back to you.

**Comment by aisamuel** [comment]
Ok harryanand060, I will be expecting. I will love to make use of your work. Thanks

**Comment by harryanand060** [comment]
Hi aisamuel, 
I checked the code and i found there is no issue. i found one issue that was some exception which i solved but this was not related to OTP issue. 

- The OTP only valid for 15 minutes. you can increase the time from local_settings **TOTP_TOKEN_VALIDITY = 900**

- OTP only valid for only one time. you can not reuse the OTP next time. 

- If you missed to get the OTP or submit the OTP  then you can only resent the the request again using **Resent API**

![image](https://user-images.githubusercontent.com/16953928/81580962-ba368200-93cb-11ea-9bd6-0a49fb2fca01.png)

please try again. if again you faced then let me know.

**Comment by aisamuel** [comment]
This same issue

**Comment by aisamuel** [comment]
![Uploading 15893846537656825399633124258247.jpg…](282620 was the otp sent to me )

**Comment by harryanand060** [comment]
I am not able to see the image.  can you send me it again?. I tried it and it was working.

**Comment by nnamdei** [comment]
Hello, 

I have the issue as above and here are the snapshot 
OTP Sent via Email
![Screenshot 2020-06-03 at 2 24 48 PM](https://user-images.githubusercontent.com/26845388/83642383-ae8f4300-a5a6-11ea-8626-d0efa5a275d6.png)
Using sent otp 
![Screenshot 2020-06-03 at 2 25 56 PM](https://user-images.githubusercontent.com/26845388/83642397-b2bb6080-a5a6-11ea-8c29-9e6905c47c68.png)
login after manually activating it in dashboard
![Screenshot 2020-06-03 at 2 26 45 PM](https://user-images.githubusercontent.com/26845388/83642403-b353f700-a5a6-11ea-9fc2-856a3f882023.png)
![Screenshot 2020-06-03 at 2 27 01 PM](https://user-images.githubusercontent.com/26845388/83642408-b4852400-a5a6-11ea-8438-833e30e514f2.png)

**Comment by harryanand060** [comment]
I don't know why you face this issue. I tried this code too many times it was working.

**Comment by nnamdei** [comment]
@harryanand060 the otp is still not working. I am using python 3.8 and dont know if that is the issue
![Screenshot 2020-08-25 at 1 42 09 AM](https://user-images.githubusercontent.com/26845388/91109909-48c9c900-e674-11ea-89ed-e8b6945b3f1b.png)

**Comment by nnamdei** [comment]
It is currently verifying the generated otp after I cloned the project and switched branch to django_v3.

**Comment by harryanand060** [comment]
yes, I already explained that the master branch is with Django 2 and another branch Django_v3 is with Django 3. when I started this project that time only Django 2 was available but now Django 3 also available. there are only a few imports different. sorry, I forgot to figure out this issue.
